packages/mongodb-verified-answers/README.md

```diff
@@ -0,0 +1,3 @@
+# MongoDB Verified Answer CLI
+
+The Verified Answer CLI imports answers specified in a yaml file into the database.
```

Comment:
Probably some more info we could put in the README (install, usage example, etc) but fine to leave minimal for now IMO